internal/service/web/address.go

```diff
@@ -0,0 +1,38 @@
+package web
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gin-gonic/gin"
+	"github.com/thirdfort/thirdfort-go-code-review/internal/models"
+	"github.com/thirdfort/thirdfort-go-code-review/internal/service"
+)
+
+func FindAddress(c *gin.Context, req *models.Transaction) (*models.TransactionResponse, error) {
+	LogAddressRequest(c, req.ID)
+	addr, err := service.FindAddress(*req.ID)
+	if err != nil {
+		panic(err)
```

Comment:
Why _panic_ here? We should return and error and have caller handle the exception? API could for instance then return a 404